#306 - K1, Getthebox, Vita

AleksandraKozubal opened the pull request:
In this pr I added blade files for K1, Getthebox and Vita that uses reusable components, also added pagination in case studies page.

[Desktop design](https://www.figma.com/design/sdPYb47iLBwsA2FQMIu5BR/Blumilk?node-id=1-4&t=pmdPPCdUgAsYsw6F-0)
[Mobile design](https://www.figma.com/design/2QnCtqnBUqCoBnoys47J9d/MOBILE?node-id=0-1&t=uM19DyTxjRgwVxfv-0)

It should close #303.

AleksandraKozubal commented on public/graphics/icons/robot.svg:
If we were using a web font, there would be no way to change the accent colors

krzysztofrewak replied:
Oh, okay.

EwelinaSkrzypacz commented on lang/pl/case_studies.php:
```diff
@@ -3,9 +3,232 @@
+                "description" => "lorem ipsum tutaj potrzebujemy zdanie uzupełniające",
+            ],
+            "challenge_2" => [
+                "title" => "Implementacja modelu przywilejów i ról",
```
```suggestion
                "title" => "Implementacja modelu uprawnień i ról",
```

EwelinaSkrzypacz commented on lang/pl/case_studies.php:
```diff
@@ -3,9 +3,232 @@
+            ],
+            "challenge_4" => [
+                "title" => "Uruchamianie dostarczanych aplikacji",
+                "description" => "bez konieczności ich instalacji w systemie operacyjnym",
```
We install apps on computers :eyes: 
I can agree that user doesn't see installation wizard - he sees only progress of download and text "Instalowanie" when the app is installed

EwelinaSkrzypacz commented on lang/pl/case_studies.php:
```diff
@@ -3,9 +3,232 @@
+    "color_palette" => "Paleta kolorów",
+    "primary" => "Kolor główny",
+    "secondary" => "Kolor dodatkowy",
+    "surface_container" => "Powierzchnia",
```
I'm not a big fan of this word.

EwelinaSkrzypacz commented on public/images/case-studies/k1/example.gif:
If we want to add gif, I believe we should made better gif for that. Also the filename could be better, because this gif show application that students have installed on university computers or their private - it's some kind of client needed to install apps

EwelinaSkrzypacz reviewed (commented):
1. Why here we have dot?

![image](https://github.com/user-attachments/assets/cc0ab14b-0956-4a99-afbb-fe7f3c92bca2)

2. I have the same picture here - it's because of seeder?

![image](https://github.com/user-attachments/assets/d52ffcc4-cf0c-42b9-a3bc-6a379cdf3aa0)

3. It's okay for us that on case study lists the colored words are different that colored words in show case study?

4. Maybe we should change name "Case study" to "Nasze projekty"? :eyes: 

5. The bubbles cover the title a bit on mobile

![image](https://github.com/user-attachments/assets/72951a89-028f-41af-91ea-329f8c382576)

6. As I understand the design on mobile is still in progress? Because now case study show doesn't look like it has RWD.

Like here

![image](https://github.com/user-attachments/assets/edf22a64-e69c-4817-9724-d025e3ef834d)

AleksandraKozubal commented:
> 1. Why here we have dot?
> 
> 
> [image](https://private-user-images.githubusercontent.com/56546832/355806343-cc0ab14b-0956-4a99-afbb-fe7f3c92bca2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMwOTg5MDMsIm5iZiI6MTcyMzA5ODYwMywicGF0aCI6Ii81NjU0NjgzMi8zNTU4MDYzNDMtY2MwYWIxNGItMDk1Ni00YTk5LWFmYmItZmU3ZjNjOTJiY2EyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODA4VDA2MzAwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJmZGVkNzZjMGRkYTIwYTRkZTNlYTIxZWNlOTcwZWRhMDJjODFjMGU4OTljOTlmY2RlNDFjNzM1ZGI0MDBiNGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.YK8kHGVdZwQxXtzTa5jbY-IeKdUcMiuA7tNQoyCvseY)
> 
> 2. I have the same picture here - it's because of seeder?
> 
> 
> [image](https://private-user-images.githubusercontent.com/56546832/355808429-d52ffcc4-cf0c-42b9-a3bc-6a379cdf3aa0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMwOTg5MDMsIm5iZiI6MTcyMzA5ODYwMywicGF0aCI6Ii81NjU0NjgzMi8zNTU4MDg0MjktZDUyZmZjYzQtY2YwYy00MmI5LWEzYmMtNmEzNzljZGYzYWEwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODA4VDA2MzAwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAwNjIxMmExNzAwNmM1M2Q1Mzk5ZDlhMzI3YWY2Yzc5ZDljN2U2OWU3OGFjZTE3ZjZhYjY0MDZmZGY5YWQ2YTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0._DS-qa-WZ2351zvzsUMpuPISLTbQrglIjJSD8Z6OeEY)
> 
> 3. It's okay for us that on case study lists the colored words are different that colored words in show case study?
> 
>4. Maybe we should change name "Case study" to "Nasze projekty"? 👀
> 
>5. The bubbles cover the title a bit on mobile
> 
> 
> [image](https://private-user-images.githubusercontent.com/56546832/355809081-72951a89-028f-41af-91ea-329f8c382576.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMwOTg5MDMsIm5iZiI6MTcyMzA5ODYwMywicGF0aCI6Ii81NjU0NjgzMi8zNTU4MDkwODEtNzI5NTFhODktMDI4Zi00MWFmLTkxZWEtMzI5ZjhjMzgyNTc2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODA4VDA2MzAwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA5YzhiY2RmYzAyMjZkN2FkZDc2OTc5YzhmYjg0ZWRjZjYyN2YyZTFkNDNkOTZmMTFhZWNhMmFmYmNlNjJkMmEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ZkLVCtDS7MtYxzVLD32VzgONvhC_DYJ0xy7x3cNAgYg)
> 
>6. As I understand the design on mobile is still in progress? Because now case study show doesn't look like it has RWD.
> 
> 
> Like here
> 
> [image](https://private-user-images.githubusercontent.com/56546832/355809975-edf22a64-e69c-4817-9724-d025e3ef834d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMwOTg5MDMsIm5iZiI6MTcyMzA5ODYwMywicGF0aCI6Ii81NjU0NjgzMi8zNTU4MDk5NzUtZWRmMjJhNjQtZTY5Yy00ODE3LTk3MjQtZDAyNWUzZWY4MzRkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODA4VDA2MzAwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWEzZDllOGMyYmViZDVmMTIyNDBiOGU5YWE5OGNkY2Q3N2QxNTY2MWY5YmQwY2M0OGJjZTQ0MzEyNjBlNTA5MzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.GJKk5aQxC-FyAkeI50UCD1F0Os-mpYe9aftfmeEGlHc)

1.It was like that in design
2.That's right, we use one image to decrease seeder's time
3.I think so, because case study tile is meant to be created from admin panel, so admin can choose how is the text colored. Now i just did same as in design.
4.Changed to "Projekty"
5.fixed
6.Yes, we don't have design yet

EwelinaSkrzypacz reviewed (commented):
1. For me this sitemap is too small (I think that on design is bigger)

![image](https://github.com/user-attachments/assets/22937dba-05f8-4040-b58e-2755b742e7a0)

2. Padding too small

![image](https://github.com/user-attachments/assets/68f033cd-2bec-4140-82e7-85c44ef392e6)

And maybe instead of "rozdział" the word "podział" will be better?

3. This picture looks different on design

![image](https://github.com/user-attachments/assets/948152ec-5949-47f8-9289-5693e23d45e7)

4. For me it should be "Kolor podstawowy" and "Kolor dodatkowy"

![image](https://github.com/user-attachments/assets/f23ef295-1d18-47bd-85b7-3c014017533b)

AleksandraKozubal commented:
> 1. For me this sitemap is too small (I think that on design is bigger)
> 
> 
> [image](https://private-user-images.githubusercontent.com/56546832/359017453-22937dba-05f8-4040-b58e-2755b742e7a0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQwNTU5MzksIm5iZiI6MTcyNDA1NTYzOSwicGF0aCI6Ii81NjU0NjgzMi8zNTkwMTc0NTMtMjI5MzdkYmEtMDVmOC00MDQwLWI1OGUtMjc1NWI3NDJlN2EwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE5VDA4MjAzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRhNzczMTExYjA4MjdiN2Q2ZjMwNDRkNTEzYTkxMTBjMzgyYzU3ZTQ0MzkyNzdmMTgxNzhmNDM2NDhhMjlkYjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.M1xTcekpAQIkL_jHEwETu9i3mqd-YotzR3pEfPbwWVo)
> 
> 2. Padding too small
> 
> 
> [image](https://private-user-images.githubusercontent.com/56546832/359017941-68f033cd-2bec-4140-82e7-85c44ef392e6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQwNTU5MzksIm5iZiI6MTcyNDA1NTYzOSwicGF0aCI6Ii81NjU0NjgzMi8zNTkwMTc5NDEtNjhmMDMzY2QtMmJlYy00MTQwLTgyZTctODVjNDRlZjM5MmU2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE5VDA4MjAzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg2Y2MxZjc5NTJlYzIzMTI4NjhmYTg1OTc4M2RhNmVkMmRjOWFmODVhOWZmMzlmNGE0ODc5ZjdhNWRlY2FiMDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Pun8A25CACp7dwZeLq19422Acu5aTBRZEN9p048IyLM)
> 
> And maybe instead of "rozdział" the word "podział" will be better?
> 
>3. This picture looks different on design
> 
> 
> [image](https://private-user-images.githubusercontent.com/56546832/359018447-948152ec-5949-47f8-9289-5693e23d45e7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQwNTU5MzksIm5iZiI6MTcyNDA1NTYzOSwicGF0aCI6Ii81NjU0NjgzMi8zNTkwMTg0NDctOTQ4MTUyZWMtNTk0OS00N2Y4LTkyODktNTY5M2UyM2Q0NWU3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE5VDA4MjAzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc1MjJjOWUxYWJjOTI4ZjU2NTllZGQwNmFiNjZjYTRjZDMzZTI0MjdlOTk2NGE4YjIyOTRmYzY3NzBjZDE2NzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.uh-hNSMp_kbYocLW8HvLam42pZSbJSuFFWCcM6yXBY4)
> 
> 4. For me it should be "Kolor podstawowy" and "Kolor dodatkowy"
> 
> 
> [image](https://private-user-images.githubusercontent.com/56546832/359018546-f23ef295-1d18-47bd-85b7-3c014017533b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQwNTU5MzksIm5iZiI6MTcyNDA1NTYzOSwicGF0aCI6Ii81NjU0NjgzMi8zNTkwMTg1NDYtZjIzZWYyOTUtMWQxOC00N2JkLTg1YjctM2MwMTQwMTc1MzNiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE5VDA4MjAzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkxOWY2NjUwOWQxMjU1N2Y2ZDI1ODJjMTMzY2JjNGVjYzcyOTdkMDRiZDM5MDVhOTRmZTlmM2FkYzc0Y2E1ZDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.7pi9oHx3RC-Ch6fqnaLZ8Wur68KQhDxxWGQmdgOnowQ)

1,2,3,4. fixed

krzysztofrewak commented on app/Http/Controllers/CaseStudiesController.php:
```diff
@@ -24,7 +24,10 @@ public function get(Factory $factory, string $slug): View
 
         $view = preg_replace('/\.blade\.php$/', "", $caseStudy->template);
```
Wouldn't `Str::replaceLast('.blade.php', '', $caseStudy->template);` more verbose?

krzysztofrewak replied:
Or maybe it would be better to store template name only in database?

krzysztofrewak commented on app/Http/Controllers/CaseStudiesController.php:
```diff
@@ -24,7 +24,10 @@ public function get(Factory $factory, string $slug): View
 
+        $locale = app()->getLocale();
+
         return $factory->make("case-studies/$view")
```
Don't you want to check if this file exists?

AleksandraKozubal replied:
Added

krzysztofrewak commented on routes/web.php:
```diff
@@ -21,8 +21,8 @@
 $router->get($uri->translate("about"), AboutController::class)->name("about");
-$router->get($uri->translate("case-studies"), [CaseStudiesController::class, "index"])->name("case-studies");
-$router->get($uri->translate("case-studies/{slug}"), [CaseStudiesController::class, "get"])->name("case-studies.entry");
+$router->get($uri->translate("projects"), [CaseStudiesController::class, "index"])->name("projects");
```
I would like to unify naming: projects or case studies.

AleksandraKozubal replied:
Done
